### PR TITLE
Crawling sounds

### PR DESCRIPTION
**Pull request by Bm0n**
## What Does This PR Do

Turns on crawling sounds of which its sound file was already in our repo from when we ported footstep.dm

Crawling didn't exist when footstep.dm got ported so it was left out.

Implementation matches TG pre-refactored code.
## Why It's Good For The Game

I like little details like this and it kind of looks weird to watch people silently slither across the floor.
## Images of changes

Turn on sound.

[Unknown 2022.12.10-16.31-1.webm](https://user-images.githubusercontent.com/92271472/206876011-900c21ef-d6d4-4816-b5d3-258501296606.webm)

## Testing

I crawled around a bunch
## Changelog
:cl: Bmon
soundadd: There is now a step sound for when you crawl.
/:cl:

### CODE REVIEW

File: code/datums/components/footstep.dm [hunk 1]
```diff
@@ -53,7 +53,11 @@
 		return
 
 	var/mob/living/LM = parent
-	if(!T.footstep || IS_HORIZONTAL(LM) || !(LM.mobility_flags & MOBILITY_MOVE) || LM.buckled || LM.throwing || LM.flying || istype(LM.loc, /obj/machinery/atmospherics))
+	if(!T.footstep || !(LM.mobility_flags & MOBILITY_MOVE) || LM.buckled || LM.throwing || LM.flying || istype(LM.loc, /obj/machinery/atmospherics))
+		return
+
+	if(LM.body_position == LYING_DOWN) //play crawling sound if we're lying
```
Comment (hal9000PR) [inline code/datums/components/footstep.dm:59]: Use the IS_HORIZONTAL define

**Comment by S34NW** [comment]
Sounds a little like too much like an industrial machine or something, but you didn't add the sound so that's not on you.

**Comment by Bm0n** [comment]
> Sounds a little like too much like an industrial machine or something, but you didn't add the sound so that's not on you.

Man, I was looking around on freesounds and I really don't think there's anything else better. You need to have that whoosh sound of your clothes and yourself getting dragged along the floor and it needs to have a noticeable stop.

**Comment by Scribble-Sheep** [comment]
> > Sounds a little like too much like an industrial machine or something, but you didn't add the sound so that's not on you.
> 
> Man, I was looking around on freesounds and I really don't think there's anything else better. You need to have that whoosh sound of your clothes and yourself getting dragged along the floor and it needs to have a noticeable stop.

I think the industrial machine-ness of it is less the sound itself, and more of how rapid it is. The gap is a bit too short compared to like, the cadence of an actual person crawling. With the current speed, it just sounds like someone rubbing something together?

**Comment by S34NW** [comment]
Yeah, it's like a slow moving steam train or something at the minute. Maybe some delay would help? I'm not sure.

**Comment by Contrabang** [comment]
> Yeah, it's like a slow moving steam train or something at the minute. Maybe some delay would help? I'm not sure.

This is gonna be terrifying to hear as the 4 random assistants crawl down the hallway.

Also, does this sound play when you drag someone whose lying down?

**Comment by Bm0n** [comment]
> This is gonna be terrifying to hear as the 4 random assistants crawl down the hallway.
> 
> Also, does this sound play when you drag someone whose lying down?

Keep in mind the sound falloff this has, it's the same as footsteps

And yes it does

**Comment by Bm0n** [comment]
Plays every second step now just like normal footsteps

[Unknown 2022.12.10-16.31-1.webm](https://user-images.githubusercontent.com/92271472/206876011-900c21ef-d6d4-4816-b5d3-258501296606.webm)

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**